rotkehlchen/assets/converters.py

```diff
@@ -354,13 +356,15 @@
     'BST',  # No coingecko or cryptocompare yet. Beshare Token TODO: Review this one in a few days
     'CADX',  # no cryptocompare/coingecko data TODO: Review this one
     'CBC',  # neither in coingecko nor cryptocompare
+    'CIND',  # neither in coingecko nor cryptocompare
     # Tether CNH. As of 30/09/2019 no data found outside of Bittrex for this token
     # https://medium.com/bittrex/new-bittrex-international-listing-tether-cnh-cnht-c9ad966ac303
     'CNHT',
     # Credit coin. As of 29/01/2020 no data found outside of Bittrex for this token
     # https://global.bittrex.com/Market/Index?MarketName=BTC-CTC
     'CTC',
     'DAF',  # neither in coingecko nor cryptocompare
+    'DATA',  # Couldn't find what token this is
```

Comment:
Didn't you say it's DAT after all?

Reply:
Ah I have to remove it. Yes omw

Reply:
This is for BITTREX and by searching in their markets they don't have any DATA token or `streamr` that could be the candidate. Searched again for posts and tweets but there is nothing